include/vcpkg/commands.acquire.h

```diff
@@ -1,11 +1,9 @@
 #pragma once
 
-#include <vcpkg/commands.interface.h>
+#include <vcpkg/fwd/vcpkgcmdarguments.h>
+#include <vcpkg/fwd/vcpkgpaths.h>
```

Comment:
Should `vcpkgcmdarguments.h` also be `vcpkg-cmd-arguments.h` for consistency? (same for `vcpkgpaths.h`)
If not, when should file names get the "-" between words and when should it just be one word?

Reply:
The result of the polls yesterday was only about command names, that the name of the file should match the name of the command. This is why `commands.portsdiff.h` and `commands.set-installed.h` -- the dash is in the command name in the second case but not the first.

I don't have a strong opinion about `vcpkgcmdarguments.h`. I was really mostly going for consistency here and I don't think that name is consistent or inconsistent with anything.